app/scripts/modules/core/src/cloudProvider/skin.service.ts

```diff
@@ -86,6 +87,27 @@ export class SkinService {
     });
   }
 
+  public static getMonikerForInstance(cloudProvider: string, instanceId: string, app: Application): IPromise<IMoniker> {
```

Comment:
I'm not entirely sure whether or not we need to look through all these server groups and load balancers to find moniker data like we're currently doing with accounts, so would be cool to get some more context 🤷‍♂️ 

Maybe it's actually the same for this as it is for accounts and we really need this?

Reply:
@imosquera might be able to lend some insight

Reply:
would love it if @imosquera or anyone else with moniker knowledge could weigh in, _but_ the worst case here is doing a tiny bit of unnecessary looping/work so I'm going to merge for now and amend this method later if new info arises 🚀